poincare/src/trigonometry_cheat_table.cpp

```diff
@@ -82,7 +82,7 @@ Expression TrigonometryCheatTable::simplify(const Expression e, ExpressionNode::
  * For instance, when simplfy a Cosine, we always compute the value for an angle
  * in the top right trigonometric quadrant. */
 const TrigonometryCheatTable * TrigonometryCheatTable::Table() {
-  static Row sTableRows[] = {
+  static const Row sTableRows[] = {
```

Comment:
```suggestion
  static Row sTableRows[] = {
```

I think the `const` makes crash some operations like `e^i.pi` or `cos(0)`

Reply:
I could not reproduce this!

Reply:
My bad, it was because of this : https://github.com/Omega-Numworks/Omega/commit/f763563834be83ffae397e047191a86fc59703b3#diff-5cf7de266c42fa98c531e899b90125e8 ; everything works perfectly